osbuild/testutil/__init__.py

```diff
@@ -67,6 +67,14 @@ def finder(s): return expected_err in s  # pylint: disable=C0321
                for err_msg in err_msgs), f"{expected_err} not found in {err_msgs}"
 
 
+def assert_equal(a, b):
+    assert a == b, f"Equality check failed: '{a}' == '{b}'"
+
+
+def assert_file_exists(f: pathlib.Path):
```

Comment:
Also here I'd love the see the path if it's failing as usually the path is some variable and "assert False" doesn't really help when inside some logs.
I'll check again if it would show in the lines above as in your example - I thought that was not the case on my end